modules/react-components/src/components/header/header.tsx

```diff
@@ -211,274 +217,274 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (
+                                                        ? userDropdownLinks.map((link, index) => {
+                                                            const {
+                                                                content,
+                                                                icon,
+                                                                name,
+                                                                onClick
+                                                            } = link;
 
-                                                      return (
-                                                          <Dropdown.Item
-                                                              key={ index }
-                                                              className="action-panel"
-                                                              onClick={ onClick }
-                                                              data-testid={ `${ testId }-dropdown-link-${ name.replace(" ", "-") }` }
-                                                          >
-                                                              {
-                                                                  icon &&
-                                                                  <Icon
-                                                                      className="link-icon"
-                                                                      name={ icon }
-                                                                  />
-                                                              }
-                                                              { name }
-                                                              { content }
-                                                          </Dropdown.Item>
-                                                      );
-                                                  })
-                                                  : null
-                                          }
-                                      </Dropdown.Menu>
-                                  }
-                              </Dropdown>
-                          )
-                      }
-                  </Menu.Menu>
-              ) }
-          </Container>
-          { children }
-      </Menu>
+                                                            return (
+                                                                <Dropdown.Item
+                                                                    key={ index }
+                                                                    className="action-panel"
+                                                                    onClick={ onClick }
+                                                                    data-testid={
+                                                                        `${ testId }-dropdown-link-${
```

Comment:
Do we need to enforce the replacement of  " " to "-" here?

Reply:
yes. The `name` here can contain `spaces`. This block was only reformatted in this PR.